### PR TITLE
mysql: note limitations of secure connections and MySQL 5.6

### PR DESCRIPTION
**Pull request by blgm**
[#184048087](https://www.pivotaltracker.com/story/show/184048087)

Which other branches should this be merged with (if any)?
- none

### CODE REVIEW

File: reference/gcp-mysql.html.md.erb [hunk 1]
```diff
@@ -290,7 +291,11 @@ provisioning a csb-google-mysql service:
   <tr>
     <td><code>allow_insecure_connections</code></td>
     <td>boolean</td>
-    <td>Specifies whether insecure connections are allowed for this service instance.</td>
+    <td>
+      Specifies whether insecure connections are allowed for this service instance.
+      Note that secure connections are not supported by MySQL 5.6 and earlier, so it's necessary
```
Comment (jhvhs) [inline reference/gcp-mysql.html.md.erb:296]: suggestion: I would re-word this as 
"Note that enforcing secure connections isn't not supported by MySQL 5.6 and earlier..." Secure connections to 5.6 are still possible, they just can't be enforced on the server-side.

File: reference/gcp-mysql.html.md.erb [hunk 2]
```diff
@@ -125,7 +125,8 @@ provisioning a csb-google-mysql service:
     <td>string</td>
     <td>
       Required unless defined in plan. This is the version for the MySQL instance, and it can be any
-      version that the provider supports.
+      version that the provider supports. Note that secure connections are only supported with MySQL 5.7
```
Comment (jhvhs) [inline reference/gcp-mysql.html.md.erb:128]: suggestion: I would re-word this as 
"Note that enforcing secure connections isn't not supported by MySQL 5.6 and earlier..." Secure connections to 5.6 are still possible, they just can't be enforced on the server-side.

File: reference/gcp-mysql.html.md.erb [hunk 3]
```diff
@@ -125,7 +125,8 @@ provisioning a csb-google-mysql service:
...
+      and higher, so if using MySQL 5.6 and earlier it is necessary to also set the <code>allow_insecure_connections</code> property.
```
Comment (jhvhs) [inline reference/gcp-mysql.html.md.erb:129]: suggestion: CloudSQL does not and never will support versions earlier than 5.6, so we may not need the "and earlier" here.

File: reference/gcp-mysql.html.md.erb [hunk 4]
```diff
@@ -290,7 +291,11 @@ provisioning a csb-google-mysql service:
   <tr>
     <td><code>allow_insecure_connections</code></td>
     <td>boolean</td>
-    <td>Specifies whether insecure connections are allowed for this service instance.</td>
+    <td>
+      Specifies whether insecure connections are allowed for this service instance.
```
Comment (jhvhs) [inline reference/gcp-mysql.html.md.erb:295]: Same as above

Comment (blgm) [reply reference/gcp-mysql.html.md.erb:288]: Updated

File: reference/gcp-mysql.html.md.erb (hunk 3 above)
Comment (blgm) [reply reference/gcp-mysql.html.md.erb]: Updated

File: reference/gcp-mysql.html.md.erb (hunk 2 above)
Comment (blgm) [reply reference/gcp-mysql.html.md.erb]: Updated

**Review by jhvhs — APPROVED** [review]

**Pull request merged**